app/experimenter/experiments/serializers.py

```diff
@@ -703,3 +718,41 @@ class ExperimentDesignGenericSerializer(ExperimentDesignBaseSerializer):
     class Meta:
         model = Experiment
         fields = ("type", "design", "variants")
+
+
+class ExperimentBranchedAddonBranchSerializer(ExperimentDesignBranchBaseSerializer):
+    addon_release_url = serializers.URLField(max_length=400)
+
+    class Meta:
+        model = ExperimentVariant
+        fields = ["addon_release_url", "id", "description", "is_control", "name", "ratio"]
+
+
+class ExperimentDesignBranchedAddonSerializer(serializers.ModelSerializer):
+    variants = ExperimentBranchedAddonBranchSerializer(many=True)
+    is_branched_addon = serializers.BooleanField()
+
+    class Meta:
+        model = Experiment
+        fields = ("type", "is_branched_addon", "variants")
+
+    def update(self, instance, validated_data):
```

Comment:
Can you inherit from ExperimentDesignBaseSerializer here and not need to rewrite update?